.gitignore

```diff
@@ -2,5 +2,6 @@
 work/
 data/
 results/
-tests/test_data/ngi-bisulfite_test_set/
 .DS_Store
+tests/test_data
+*.pyc
```

Comment:
same

Reply:
Disagree. These files are created automatically when running python scripts and should not be committed. Any project that has any python code should have this in `.gitignore` (and again, doesn't do any harm..?)

Reply:
Should you be running python script here?
But agree, doesn't do any harm.

Reply:
I guess not, but whilst developing pipelines it's not uncommon to be messing around with the bundled python scripts (eg. version script). I think that's probably why it was added..